### PR TITLE
Roostats formatting

### PR DESCRIPTION
**Pull request by couet**
- spell check
- indentation
- comment ins doxygen format
- remove dead code

### CODE REVIEW

File: roofit/roostats/inc/RooStats/ProfileLikelihoodTestStat.h [hunk 1]
```diff
@@ -156,13 +150,13 @@ namespace RooStats {
       LimitType fLimitType;
       Bool_t fSigned;
 
-      // this will store a snapshot of the unconditional nuisance
-      // parameter fit.
+      /// this will store a snapshot of the unconditional nuisance
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/ProfileLikelihoodTestStat.h:153]: This comment can be removed. It is not documenting the following members

File: roofit/roostats/inc/RooStats/RooStatsUtils.h [hunk 2]
```diff
@@ -58,9 +58,9 @@ namespace RooStats {
    }
 
    /// Compute the Asimov Median significance for a Poisson process
-   /// with s = expected number of signal events, b = expected numner of background events
-   /// and optionally sigma_b = expected uncertainty of backgorund events   
-   Double_t AsimovSignificance(Double_t s, Double_t b, Double_t sigma_b = 0.0 ); 
+   /// with s = expected number of signal events, b = expected number of background events
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/RooStatsUtils.h:61]: /// param[in] s = expected number of signal events
/// param[in] b = expected number of background events
/// param[in] sigma_b : expected uncertainty of background events. A default value of 0 is used if it is not given.

File: roofit/roostats/inc/RooStats/RooStatsUtils.h [hunk 3]
```diff
@@ -86,9 +86,9 @@ namespace RooStats {
       set.remove(constSet);
    }
 
+   /// utility function to set all variable constant in a collection
+   /// (from G. Petrucciani)
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/RooStatsUtils.h:90]: This citation can be removed from the visible doc

File: roofit/roostats/inc/RooStats/SimpleInterval.h [hunk 4]
```diff
@@ -14,57 +14,59 @@
...
 
   public:
-    // default constructors
+    /// default constructor
     explicit SimpleInterval(const char* name = 0);
 
+    /// default constructor
     SimpleInterval(const SimpleInterval& other, const char* name);
 
+    /// default constructor
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/SimpleInterval.h:29]: /// Assignment operator

File: roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h [hunk 5]
```diff
@@ -25,30 +25,27 @@ namespace RooStats {
...
-    fReuseNll=kFALSE ;
-    fNllNull=NULL ;
-    fNllAlt=NULL ;
+         fReuseNll=kFALSE ;
+         fNllNull=NULL ;
+         fNllAlt=NULL ;
       }
 
-      //__________________________________
+      /// Takes null and alternate parameters from PDF. Can be overridden.
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h:42]: /// Constructor passing PDF for the null and the alternate models. 
/// Take the corresponding parameters from the pdf's.

File: roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h [hunk 6]
```diff
@@ -63,11 +60,13 @@ namespace RooStats {
...
-    fReuseNll=kFALSE ;
-    fNllNull=NULL ;
-    fNllAlt=NULL ;
+         fReuseNll=kFALSE ;
+         fNllNull=NULL ;
+         fNllAlt=NULL ;
       }
-      //__________________________________
+
+      /// Takes null and alternate parameters from values in nullParameters
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h:68]: /// Constructor passing PDF for the null and the alternate models and 
///  the corresponding model parameters

File: roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h [hunk 7]
```diff
@@ -87,43 +84,37 @@ namespace RooStats {
...
-      //_________________________________________
       void SetAltParameters(const RooArgSet& altParameters) {
          if (fAltParameters) delete fAltParameters;
          fFirstEval = true;
-         //      if(fAltParameters) delete fAltParameters;
          fAltParameters = (RooArgSet*) altParameters.snapshot();
       }
 
-      //______________________________
+      /// this should be possible with RooAbsCollection
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h:116]: /// Check if null and alt parameters are equal

File: roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h [hunk 8]
```diff
@@ -170,10 +160,10 @@ namespace RooStats {
       bool fFirstEval;
 
       bool fDetailedOutputEnabled;
-      RooArgSet* fDetailedOutput; //!
+      RooArgSet* fDetailedOutput; ///<!
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/SimpleLikelihoodRatioTestStat.h:163]: ///<! Transient pointer to the detailed output

File: roofit/roostats/inc/RooStats/TestStatSampler.h [hunk 9]
```diff
@@ -39,52 +39,48 @@ used for coverage studies, the Neyman Construction, etc.
    class TestStatSampler {
 
    public:
-     //     TestStatSampler();
-     virtual ~TestStatSampler() {}
+      virtual ~TestStatSampler() {}
 
-      // Main interface to get a ConfInterval, pure virtual
+      /// Main interface to get a ConfInterval, pure virtual
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/TestStatSampler.h:44]: /// Get sampling distribution

File: roofit/roostats/inc/RooStats/ToyMCImportanceSampler.h [hunk 10]
```diff
@@ -45,9 +45,9 @@ class ToyMCImportanceSampler: public ToyMCSampler {
 
       virtual ~ToyMCImportanceSampler();
 
-      // overwrite GetSamplingDistributionsSingleWorker(paramPoint) with a version that loops
-      // over nulls and importance densities, but calls the parent
-      // ToyMCSampler::GetSamplingDistributionsSingleWorker(paramPoint).
+      /// overwrite GetSamplingDistributionsSingleWorker(paramPoint) with a version that loops
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/ToyMCImportanceSampler.h:48]: /// Overwritten implementation of GetSamplingDistributionsSingleWorker(paramPoint) ....

File: roofit/roostats/inc/RooStats/ToyMCSampler.h [hunk 11]
```diff
@@ -76,7 +76,7 @@ class ToyMCSampler: public TestStatSampler {
 
       void SetUseMultiGen(Bool_t flag) { fUseMultiGen = flag ; }
 
-      // main interface
+      /// main interface
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/ToyMCSampler.h:79]: /// Interface to get a sampling distribution

File: roofit/roostats/inc/RooStats/ToyMCSampler.h [hunk 12]
```diff
@@ -147,65 +146,65 @@ class ToyMCSampler: public TestStatSampler {
...
-      // Set the Pdf, add to the the workspace if not already there
+      /// Set the Pdf, add to the the workspace if not already there
       virtual void SetParametersForTestStat(const RooArgSet& nullpoi) {
          fParametersForTestStat.reset( nullpoi.snapshot() );
       }
 
       virtual void SetPdf(RooAbsPdf& pdf) { fPdf = &pdf; ClearCache(); }
 
-      // How to randomize the prior. Set to NULL to deactivate randomization.
+      /// How to randomize the prior. Set to NULL to deactivate randomization.
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/ToyMCSampler.h:156]: /// Set a prior pdf to randomize nuisance parameters. 
/// Set to NULL to de-activate parameter randomisation

File: roofit/roostats/inc/RooStats/ToyMCSampler.h [hunk 13]
```diff
@@ -147,65 +146,65 @@ class ToyMCSampler: public TestStatSampler {
...
+         else
+            fTestStatistics[i] = testStatistic;
       }
       virtual void SetTestStatistic(TestStatistic *t) { return SetTestStatistic(t,0); }
 
       virtual void SetExpectedNuisancePar(Bool_t i = kTRUE) { fExpectedNuisancePar = i; }
       virtual void SetAsimovNuisancePar(Bool_t i = kTRUE) { fExpectedNuisancePar = i; }
 
-      // Checks for sufficient information to do a GetSamplingDistribution(...).
+      /// Checks for sufficient information to do a GetSamplingDistribution(...).
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/ToyMCSampler.h:193]: /// Checks for a correct configuration before using GetSamplingDistribution

File: roofit/roostats/inc/RooStats/ToyMCStudy.h [hunk 14]
```diff
@@ -30,7 +30,7 @@ namespace RooStats {
 class ToyMCStudy: public RooAbsStudy {
 
    public:
-      // need to have constructor without arguments for proof
+      /// need to have constructor without arguments for proof
```
Comment (lmoneta) [inline roofit/roostats/inc/RooStats/ToyMCStudy.h:33]: /// Default constructor

**Review by lmoneta — APPROVED** [review]
Thank you very much Olivier for going through all this code and fix all these Doxygen issues!

I have added some additional fixes looking at their correctness. If there are too many to implement I can include them in a following PR. 

One other general comment is that the comments documenting the member functions will be better if they start with a capital letter. Also this could be done in a following PR.

File: roofit/roostats/inc/RooStats/BayesianCalculator.h [hunk 15]
```diff
@@ -50,23 +50,23 @@ namespace RooStats {
...
-      // destructor
+      /// destructor
       virtual ~BayesianCalculator();
 
-      // get the plot with option to get it normalized
+      /// get the plot with option to get it normalized
       RooPlot* GetPosteriorPlot(bool norm = false, double precision = 0.01) const;
 
-      // return posterior pdf (object is managed by the user)
+      /// return posterior pdf (object is managed by the user)
```
[the remark this answers is not in the document]
Comment (couet) [reply roofit/roostats/inc/RooStats/BayesianCalculator.h:59]: Yes may be but I did not touch the comments yet unless there was a spelling mistake. I concentrate on the doxygen formatting and make sure the doc is visible ... and visible in the right place.

**Comment by guitargeek** [comment]
Just refreshing the tests before merging:

**Comment by guitargeek** [comment]
@phsft-bot build

**Pull request merged**